sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,300 @@
+| [`ALTER USER`](/sql-statements/sql-statement-alter-user.md) | Modifies a user. |
+| [`CREATE ROLE`](/sql-statements/sql-statement-create-role.md) | Creates a role. |
+| [`CREATE USER`](/sql-statements/sql-statement-create-user.md) | Creates a new user. |
+| [`DROP ROLE`](/sql-statements/sql-statement-drop-role.md) | Drops an existing role. |
+| [`DROP USER`](/sql-statements/sql-statement-drop-user.md) | Drops an existing user. |
+| [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md) | Activates privilege changes. |
+| [`GRANT <privileges>`](/sql-statements/sql-statement-grant-privileges.md) | Grants privileges. |
+| [`GRANT <role>`](/sql-statements/sql-statement-grant-role.md) | Grants a role. |
+| [`RENAME USER`](/sql-statements/sql-statement-rename-user.md) | Renames an existing user. |
+| [`REVOKE <privileges>`](/sql-statements/sql-statement-revoke-privileges.md) | Revokes privileges. |
+| [`REVOKE <role>`](/sql-statements/sql-statement-revoke-role.md) | Revokes a role. |
+| [`SET DEFAULT ROLE`](/sql-statements/sql-statement-set-default-role.md) | Sets a default role. |
+| [`SET PASSWORD`](/sql-statements/sql-statement-set-password.md) | Changes a password. |
+| [`SET ROLE`](/sql-statements/sql-statement-set-role.md) | Sets a role. |
+| [`SHOW CREATE USER`](/sql-statements/sql-statement-show-create-user.md) | Shows the CREATE statement for a user. |
+| [`SHOW GRANTS`](/sql-statements/sql-statement-show-grants.md) | Shows grants for a user. |
+| [`SHOW PRIVILEGES`](/sql-statements/sql-statement-show-privileges.md) | Shows available privileges. |
+
+<CustomContent platform="tidb">
+
+## TiDB Binlog & TiCDC
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ADMIN [SET\|SHOW\|UNSET] BDR ROLE`](/sql-statements/sql-statement-admin-bdr-role.md) | Manages BDR roles. |
+| [`CHANGE DRAINER`](/sql-statements/sql-statement-change-drainer.md) | Changes drainer. |
+| [`CHANGE PUMP`](/sql-statements/sql-statement-change-pump.md) | Changes pump. |
+| [`SHOW DRAINER STATUS`](/sql-statements/sql-statement-show-drainer-status.md) | Shows the status of the drainer. |
+| [`SHOW MASTER STATUS`](/sql-statements/sql-statement-show-master-status.md) | Shows the status of the master server. |
+| [`SHOW PUMP STATUS`](/sql-statements/sql-statement-show-pump-status.md) | Shows the status of the pump. |
+
+</CustomContent>
+
+## Statistics and plan management
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ANALYZE TABLE`](/sql-statements/sql-statement-analyze-table.md) | Collects statistics about a table. |
+| [`CREATE BINDING`](/sql-statements/sql-statement-create-binding.md) | Creates a plan binding. |
+| [`DROP BINDING`](/sql-statements/sql-statement-drop-binding.md) | Drops a plan binding. |
+| [`DROP STATS`](/sql-statements/sql-statement-drop-stats.md) | Drops statistics from a table. |
+| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) | Analyze query execution. |
```

Comment:
```suggestion
| [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md) | Works similar to `EXPLAIN`, with the major difference that it will execute the statement. |
```